services/core-api/tests/projects/project_summaries/resources/test_project_summary_resource.py

```diff
@@ -83,131 +92,338 @@ def test_update_project_summary_assign_project_lead(test_client, db_session, aut
-    '''Not agreeing to AMS terms when there are AMS auths returns an error'''
-    project = ProjectFactory(project_summary=0)
-    project_summary = ProjectSummaryFactory(project=project)
-    party = PartyFactory(person=True)
-
-    data = {}
     data['documents'] = []
-    data['mine_guid'] = project_summary.project.mine_guid
+
+    # Basic info data
     data['project_summary_title'] = project_summary.project_summary_title
-    data['status_code'] = 'SUB'
-    data['confirmation_of_submission'] = True
+    data['project_summary_description'] = project_summary.project_summary_description
+
+    # Project contacts data
+    data['contacts'] = [
+        {
+            "email": "test@gov.bc.ca",
+            "phone_number": "123-123-1234",
+            "is_primary": True,
+            "first_name": "test name",
+            "last_name": "test name",
+            "address": {
+                "suite_no": "123",
+                "address_line_1": "some street",
+                "city": "Vancouver",
+                "sub_division_code": "BC",
+                "post_code": "V5S4W2",
+                "address_type_code": "CAN"
+            }
+        },
+    ]
+
+    # Authorization data
     data['ams_authorizations'] = {
-        'amendments': [],
-        'new': [{
-            'authorization_description': 'Purpose of application',
-            'exemption_requested': False,
-            'new_type': 'APP',
-            'project_summary_authorization_type': 'MUNICIPAL_WASTEWATER_REGULATION',
-            'project_summary_guid': project_summary.project_summary_guid,
-            'project_summary_permit_type': ['NEW']
-        }]
+    "amendments": [
+        {
+            "project_summary_permit_type": [
+                "AMENDMENT"
+            ],
+            "project_summary_authorization_type": "AIR_EMISSIONS_DISCHARGE_PERMIT",
+            "existing_permits_authorizations": [
+                "1234"
+            ],
+            "amendment_changes": [
+                "ILT",
+                "IGT"
+            ],
+            "amendment_severity": "SIG",
+            "is_contaminated": False,
+            "authorization_description": "test descript",
+            "exemption_requested": False,
+        }
+    ],
+    "new": []
     }
-    data['project_lead_party_guid'] = party.party_guid
+
+    data['authorizations'] = [
+        {
+            "project_summary_permit_type": [
+                "AMENDMENT"
+            ],
+            "project_summary_authorization_type": "AIR_EMISSIONS_DISCHARGE_PERMIT",
+            "existing_permits_authorizations": [
+                "1234"
+            ],
+            "amendment_changes": [
+                "ILT",
+                "IGT"
+            ],
+            "amendment_severity": "SIG",
+            "is_contaminated": False,
+            "authorization_description": "description",
+            "exemption_requested": False,
+        }
+    ]
+
+    # Applicant data
+    data['applicant'] = {
```

Comment:
Some of these data sections are really beefy! Maybe wouldn't hurt to define a couple of constants like applicant_data to make things a little easier to read & write

Reply:
Some constants have been setup now for some of the data